src/mount.c

```diff
@@ -153,6 +155,18 @@ cc_oci_perform_mount (const struct cc_oci_mount *m, gboolean dry_run)
 		return true;
 	}
 
+	if (stat (m->mnt.mnt_fsname, &st) < 0) {
+		g_critical ("mount file %s does not exist", m->mnt.mnt_fsname);
+		return false;
+	}
+
+	if (S_ISREG(st.st_mode)) {
+		if(creat (m->dest, st.st_mode) < 0 ) {
+			g_critical ("failed to create mount file: %s", m->dest);
```

Comment:
Could you add a `%s` for `strerror (errno)` so the actual `errno` gets logged? You could do that for the `stat()` too. Maybe something like:

```
g_critical ("Unable to handle mount file %s: %s", m->mnt.mnt_fsname, sterror (errno));
```